lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -113,6 +119,42 @@ public void onUpdate(AtlasLibraryLoader loader) {
         });
     }
 
+    /**
+     * The user has specified an ADM file on inception.  Clear current mappings and process
```

Comment:
exra space between "inception." and "Clear"

Reply:
ok